cloudflare/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM debian:jessie
```

Comment:
I might be surprised you're using Debian as the baseimage, rather than Alpine, but I don't have any opinion on it.